src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -191,6 +183,26 @@ export const Checkout: React.FC<Props> = ({
 
   const outerWrapper = useRef<HTMLDivElement>()
 
+  const [wrapperScrollWidth, setWrapperScrollWidth] = useState<
+    number | undefined
+  >()
+
+  useEffect(() => {
+    const setScrollWidth = () => {
+      if (outerWrapper.current?.scrollWidth) {
+        setWrapperScrollWidth(outerWrapper.current?.scrollWidth)
+      }
+    }
+
+    setScrollWidth()
+
+    window.addEventListener('resize', setScrollWidth)
+
+    return () => {
+      window.removeEventListener('resize', setScrollWidth)
+    }
+  }, [outerWrapper.current?.scrollWidth])
+
```

Comment:
Didn't fully understand what issue we solved with the scroll width 🤔

Reply:
I tried to explain that in point 4. in the description but I guess I didn't do such a good job then. If the browser has a fixed scrollbar (like Edge and Explorer have, and Google had until like last weekend lol) the `Sign` component will cover that too. Which is ugly. But as I wrote in the description, if you think is just unnecessary complexity that only solves a problem for the few users who use Edge or Explorer (I mean who does, really?) then I can remove it.

Reply:
OK after looking into it more I'm not even sure it matters for Edge so I'll remove it. It was a neat solution though imo, when it was needed for Chrome last week 😄 But I'll remove it!

Reply:
It was just med that couldn't visualise it my head 😁  It think it's awesome that you took that in to consideration, I'm all for not punishing users with old browsers. But if it's not needed for Edge I agree that we can remove it to not have that complexity 🥳